Lib/functools.py

```diff
@@ -941,6 +941,9 @@ def register(self, cls, method=None):
         """
         return self.dispatcher.register(cls, func=method)
 
+    def __set_name__(self, _, name):
```

Comment:
We might find other uses for this `__set_name__` method later, in which case the name of the other parameter will be more relevant

```suggestion
    def __set_name__(self, owner, name):
```